.github/workflows/test-build-deploy.yml

```diff
@@ -92,17 +107,17 @@ jobs:
       fail-fast: false
       matrix:
         tags:
-        - requires_docker
-        - integration_alertmanager
-        - integration_backward_compatibility
-        - integration_memberlist
-        - integration_querier
-        - integration_ruler
+          - requires_docker
+          - integration_alertmanager
+          - integration_backward_compatibility
+          - integration_memberlist
+          - integration_querier
+          - integration_ruler
```

Comment:
thanks for fixing my linting bug :P. We should test lint on these yamls

Reply:
Oh I didn't fix it ... my IDE does it for me ;)